core/src/main/resources/hudson/slaves/ComputerLauncher/main.jelly

```diff
@@ -29,7 +29,9 @@ THE SOFTWARE.
       <j:when test="${it.isConnecting()}">
         <p>
           ${%launchingDescription}
+          <j:if test="${it.hasPermission(it.CONFIGURE) or it.hasPermission(it.CONNECT)}">
           <a href="log">${%See log for more details}</a>
```

Comment:
```suggestion
          <j:if test="${it.hasAnyPermission(it.CONFIGURE, it.CONNECT)}">
            <a href="log">${%See log for more details}</a>
```

Reply:
@daniel-beck , I didn't change it to suggested code, since it could not work.

Reply:
Thanks for checking 👍